Description: Change architecture from linux-any to riscv64
 This patch restricts the package to RISC-V64 architecture since we're using
 pre-built binaries specific to this architecture.
Author: Bruno Verachten <gounthar@gmail.com>
Forwarded: not-needed
Last-Update: 2025-10-18

--- a/debian/control
+++ b/debian/control
@@ -1,7 +1,7 @@
 Source: docker.io
 Section: admin
 Priority: optional
-Maintainer: Debian Go Packaging Team <team+pkg-go@tracker.debian.org>
+Maintainer: Bruno Verachten <gounthar@gmail.com>
 Build-Depends: debhelper-compat (= 13)
     ,dh-apparmor
     ,bash-completion
@@ -10,7 +10,7 @@
 Vcs-Browser: https://github.com/gounthar/docker-for-riscv64

 Package: docker.io
-Architecture: linux-any
+Architecture: riscv64
 Pre-Depends: init-system-helpers (>= 1.54~)
 Depends: ${misc:Depends}
         ,${shlibs:Depends}
